allensdk/test/model/test_biophysical_all_active.py

```diff
@@ -0,0 +1,142 @@
+# Allen Institute Software License - This software license is the 2-clause BSD
```

Comment:
You've put together some really large-scale integration tests here, but the code changes are pretty concise. Could you test these changes using a focused unit test?

Reply:
For any test we need to simulate model. That's pretty much what I am doing here.

Reply:
I agree that to really know the thing is working, you need to simulate a model and check the outputs. However, your actual code changes are much more focused and (as far as I can tell) involve changes to the code that sets up model runs, which can be tested by a small quick test independently from fitting the whole thing. Having such tests in place is important, because the large-scale integration tests are slow to run and don't provide much debugging information when they fail (because they depend on so many subcomponents). Here is a well-known [article that goes into more detail](https://testing.googleblog.com/2015/04/just-say-no-to-more-end-to-end-tests.html).

Reply:
I'm also a bit concerned about the test data you are adding. This increases the size of allensdk from 122mb -> 131mb, which is not such a big deal on its own. However, because git repositories retain all of their history, this increase in size is basically permanent, even if we replace some or all of these data down the road!

How stable do you expect these test data to be? Are they totally necessary? Could they be stripped down? If we absolutely must have them in their present form, let's come up with a way to store them that doesn't bloat our git repo.

Here is some more information on this topic:
https://github.community/t5/Support-Protips/Working-with-large-files-and-repositories/ba-p/9343

Reply:
They are not necessary if we can make an api call during the CI. There are other test files written by David where api calls are made. But as you advised against it in the previous version of the PR,  I put the data in the test folder. Do you want to git lfs? It has limited bandwidth without enrolling any plans.

Reply:
Unit tests shouldn't call the API. They:
1. can fail randomly if there is any problem making a request vs. warehouse
2. are slow, due to downloading data from warehouse

Those tests should be replaced or [marked](https://github.com/AllenInstitute/AllenSDK/blob/924757ea187948e09b29bb11f8599d4388e9bf6a/conftest.py#L59) for specific runs.

git lfs is a pretty good solution (and we actually have some storage+bandwith). I think setting it up would also require making sure our ci caches the data, so maybe not worth it as part of this work.

Another option would be to cache data on /allen, then [mark the test](https://github.com/AllenInstitute/AllenSDK/blob/924757ea187948e09b29bb11f8599d4388e9bf6a/conftest.py#L74) to only run in bamboo.

Reply:
The size should be down now. Please let me know, if it's at an acceptable stage.